modules/ocf_desktop/files/xsession/pdf-open

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+[ "$#" -ne 1 ] && { echo 'Bad arguments, trying evince...'; exec evince "$@"; }
+[ -f "$1" ] || { echo 'File does not exist, trying evince...'; exec evince "$@"; }
+
+size=$(du "$1") || { echo 'Cannot get size, trying evince...'; exec evince "$@"; }
+size=$(cut -f1 <<< "$size")
+[ "$size" -lt 5120 ] || { echo 'Too large, trying evince...'; exec evince "$@"; }
+
+echo 'Rasterizing PDF...'
+tmp=$(mktemp --suffix='.pdf')
+
```

Comment:
Since this script usually won't be run in a terminal, consider making a progress dialog using zenity. Otherwise people will wonder why their PDFs haven't opened yet and try to open it 10 times.